docs/messaging/best-practices.md

```diff
@@ -1,4 +1,62 @@
-# High Volume SMS Best Practices and Guidelines
+# Best practices for sending SMS via an API
```

Comment:
I think it would be helpful to get this commit into a different PR. That will help expedite its release. Look into creating a new branch, and then use git cherry-pick to pluck out this change and put it into this new branch